app/Http/Controllers/Auth/AuthController.php

```diff
@@ -31,13 +31,20 @@ class AuthController extends Controller
     protected $redirectTo = '/';
 
     /**
-     * Create a new authentication controller instance.
+     * The authentication guard.
      *
-     * @return void
+     * @var Guard
+     */
+    protected $guard;
+
+    /**
+     * Create a new authentication controller instance.
+     * @param Guard $guard
      */
-    public function __construct()
+    public function __construct(Guard $guard)
     {
-        $this->middleware($this->guestMiddleware(), ['except' => 'logout']);
+        $this->guard = $guard;
+        $this->middleware('role:admin', ['except' => 'logout']);
```

Comment:
Ah right! So would I have to create a `UsersController.php` that would look something similar to [this](https://github.com/DoSomething/aurora/blob/master/app/Http/Controllers/UsersController.php#L1-L27)?